app/resources/start.html.template

```diff
@@ -21,62 +21,62 @@
 {% load i18n %}
 
 {% block content %}
+  {% if env.amp %}
+    <div class="languages">
+      <a href="{{env.start_url}}">Other Languages</a>
+    </div>
+  {% endif %}
 
- {% if env.amp %}
- <div class="languages">
-   <a href="{{env.start_url}}">Other Languages</a>
- </div>
- {% endif %}
-  <div class="main">
-    <div class="row">
-      {# Following onclick is to make the whole button clickable. #}
-      <div class="action-outer start" onclick="location.href = '{{seek_url}}';" style="cursor: pointer;">
-        <div class="action-inner">
-          <a href="{{seek_url}}" tabindex="1">
-            {% trans "I'm looking for someone" %}
-          </a>
+    <div class="main">
+      <div class="row">
+        {# Following onclick is to make the whole button clickable. #}
+        <div class="action-outer start"{% if not env.amp %} onclick="location.href = '{{seek_url}}';" style="cursor: pointer;"{% endif %}>
```

Comment:
How about putting line breaks so that it's somewhat more readable?:

```html
        <div class="action-outer start"
            {% if not env.amp %}
              onclick="location.href = '{{seek_url}}';" style="cursor: pointer;"
            {% endif %}
        >
```

Same for below.